shared/dataflow/codeql/dataflow/internal/DataFlowImpl.qll

```diff
@@ -3709,16 +3725,9 @@ module MakeImpl<LocationSig Location, InputSig<Location> Lang> {
       }
 
       import CallContextSensitivity<CallContextSensitivityInput>
-      import NoLocalCallContext
+      import LocalCallContext
```

Comment:
A change like this would be nice to perf-check on its own.

Reply:
I have reverted this change.